dashboard/consts.py

```diff
@@ -9,6 +9,10 @@
 DASHBOARD_AGENT_CHECK_PARENT_INTERVAL_S = env_integer(
     DASHBOARD_AGENT_CHECK_PARENT_INTERVAL_S_ENV_NAME, 0.4
 )
+DASHBOARD_AGENT_NUM_GCS_RECONNECT_RETRY = env_integer(
+    "DASHBOARD_AGENT_NUM_GCS_RECONNECT_RETRY",
```

Comment:
should we change it to 
https://github.com/ray-project/ray/blob/master/src/ray/common/ray_config_def.h#L65C7-L65C7 ?